Library/Homebrew/extend/os/mac/formula_cellar_checks.rb

```diff
@@ -48,6 +48,31 @@ def check_openssl_links
     EOS
   end
 
+  def check_accelerate_framework_links
+    return unless @core_tap
+    return unless formula.prefix.directory?
+    return if formula.name == "veclibfort" # veclibfort exists to wrap Accelerate
+
+    keg = Keg.new(formula.prefix)
+    system_accelerate = keg.mach_o_files.select do |obj|
+      dlls = obj.dynamically_linked_libraries
+      # libBLAS and libLAPACK are superseded by functions in OpenBLAS
+      # This can be linked via multiple paths, so use a wildcard match
+      dlls.any? { |dll| %r{/System/Library/Frameworks/Accelerate.framework/.*/(libBLAS|libLAPACK).dylib$}.match dll }
+    end
+    return if system_accelerate.empty?
+
+    <<~EOS
+      object files were linked against system Accelerate
+      These object files link against outdated BLAS/LAPACK routines provided
+      by the system Accelerate framework. Core tap formulae should link
+      against OpenBLAS instead.  Removing `depends_on "veclibfort"
```

Comment:
Given we already have a separate audit for this I'm not sure what this check specifically is resolving for us. What's the actual problem users are experiencing that will be solved with this check?

Reply:
- The original audit for accelerate linkage was rolled back in #6168 and replaced with a check for dependencies on `veclibfort` and `lapack`.
- Some build systems (e.g., python packages according to @jonchang) link directly against BLAS/LAPACK dylibs within the Accelerate framework
- This audit is catching those cases.
- The actual problem would be degraded performance, and occasional build bugs due to out of date BLAS/LAPACK implementation...

FWIW, I think this check probably has marginal benefits in practice, but may still be worth doing to prevent some confusion if we end up tripping over a subtle BLAS/LAPACK bug in a formula in the future due to the antiquated implementation in Accelerate.

Reply:
(So I'm 👍 to 🚢 but won't be devastated if we don', FWIW)

Reply:
Would anyone object to holding on this for e.g. a few weeks and see if we feel like we need it then? I'd like to knowingly avoid more false positives at this point.